x-pack/test/functional/page_objects/dataset_quality.ts

```diff
@@ -104,6 +104,10 @@ export function DatasetQualityPageObject({ getPageObjects, getService }: FtrProv
       );
     },
 
+    async waitUntilTableLoaded() {
+      await testSubjects.missingOrFail('.euiBasicTable-loading');
```

Comment:
is this actually the test subject? This looks like a class selector (also judging by how it's used in the rest of the Kibana code base.

Reply:
That class gets added to the wrapper of our table whenever it's in loading state, I'd think that one is enough for our case. Do you thin more specificity will help here? Maybe something like 
```
find.byCssSelector(
    '.euiBasicTable[data-test-subj="datasetQualityTable"]:not(.euiBasicTable-loading)'
);
```

Reply:
What I'm saying is that `testSubjects` is meant to select using the `data-test-subj` attribute, not a class - I think the missingOrFail from above is checking for `[data-test-subj=".euiBasicTable-loading"]`, which is not what we want: https://github.com/elastic/kibana/blob/77964bc0f7530a4918dd00d9c0d4c0e45cdcbbf0/packages/kbn-test-subj-selector/test_subj_selector.ts#L46

To match by class name, `find.waitForDeletedByCssSelector` is probably the right method: https://github.com/elastic/kibana/blob/0515829a9e50740357c0226fd1cb5081a273ccd3/packages/kbn-ftr-common-functional-ui-services/services/find.ts#L462

Reply:
Addressed in [de3e8f2](https://github.com/elastic/kibana/pull/179440/commits/de3e8f233962d344a33178c68df769c4984f96be)